packages/debug_page/lib/src/ui/logs_inspector.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:debug_page/debug_page.dart';
+import 'package:debug_page/src/request_log.dart';
+import 'package:debug_page/src/ui/details_screen/typography.dart';
+import 'package:debug_page/src/ui/log_tile.dart';
+import 'package:flutter/material.dart';
+
+class LogsInspector extends StatelessWidget {
```

Comment:
We will definitely need to separate the logic from from there widget here. This page will not live all the time so it won't include requests which occurred when it wasn't visible

Reply:
true, I've already done it and it is coming in next PRs